cmd/configschema/go.mod

```diff
@@ -237,7 +237,7 @@ require (
 	github.com/Azure/go-autorest/logger v0.2.1 // indirect
 	github.com/Azure/go-autorest/tracing v0.6.0 // indirect
 	github.com/ClickHouse/ch-go v0.52.1 // indirect
-	github.com/ClickHouse/clickhouse-go/v2 v2.6.4 // indirect
+	github.com/ClickHouse/clickhouse-go/v2 v2.6.5 // indirect
```

Comment:
nit: Let's avoid dependency upgrades if no functionality depends on that. It'll be handled by automated depebdabot updates anyway.

Reply:
When are deps upgraded? It would be odd to revert it because there is another way.